packages/default/scss/calendar/_variables.scss

```diff
@@ -19,6 +19,7 @@ $kendo-calendar-header-bg: $kendo-component-header-bg !default;
 $kendo-calendar-header-text: $kendo-component-header-text !default;
 $kendo-calendar-header-border: $kendo-component-header-border !default;
 $kendo-calendar-header-gradient: $kendo-component-header-gradient !default;
+// TODO: variable is used, but the selector using it is not! Potentially remove;
```

Comment:
This is used in the [jQuery Classic Calendar](https://demos.telerik.com/kendo-ui/calendar/index).